templates/workspaces/base/terraform/network/security.tf

```diff
@@ -172,6 +172,23 @@ resource "azurerm_network_security_rule" "allow-inbound-from-resourceprocessor"
   source_port_range = "*"
 }
 
+
+resource "azurerm_network_security_rule" "allow-inbound-from-airlockprocessor" {
+  access                       = "Allow"
+  destination_address_prefixes = azurerm_subnet.services.address_prefixes
+  destination_port_range       = "443"
+  direction                    = "Inbound"
+  name                         = "allow-inbound-from-airlockprocessor"
+  network_security_group_name  = azurerm_network_security_group.ws.name
+  priority                     = 130
+  protocol                     = "Tcp"
+  resource_group_name          = var.ws_resource_group_name
+  source_address_prefixes = [
+    data.azurerm_subnet.airlockprocessor.address_prefix
+  ]
+  source_port_range = "*"
+}
+
```

Comment:
where is the NSG linked? Would maybe expect one for new each subnet?

Reply:
I went with making the airlock processor subnet SAME as the resource processor subnet...
the other nsg rule isnt linked anywhere as well

Reply:
Sorry, these are rules, not groups, you are correct.